### DISCUSSION TITLE
Instances still online even while host isn't

### DISCUSSION THREAD

**Discussion by pcfriek1987** [apache/cloudstack#9928] [Q&A] [answered]
A while ago we had an issue with 2 of the CS hosts where they just went offline (since we use local storage) the instances also went offline. Yes it's a deliberate choice to use local storage.

Having that said, I was expecting for those instances to go offline in the portal at some point, it knew the hosts were offline as it showed that so it should know the instances were too as it's local storage.

After the hosts were up again (nothing changed in HW or anything) it continued showing them as online, where the agent on those hosts should know they are not in fact. 

We had to actually in the portal do a "power off" so I guess it gets adjusted in the database and power them on for them to work. Is there a reason it treats it like this and how can we make sure it will just instances offline that are offline after such an outage happens?

**Comment by DaanHoogland** [comment]
@pcfriek1987 , do you have a resolution to this query yet? (or a PR/issue to remedy it?)

**Comment by pcfriek1987** [comment]
I have not as it seems to be something in cloudstack as it happens with all hosts, my best guess would be a timeout somewhere.

**Comment by DaanHoogland** [comment]
sounds bad, please keep an eye out. So in short:
- host with local storage down
- host up
- VMs still marked up while being down
Am I missing something noteworthy?

**Comment by pcfriek1987** [comment]
No that is about it

**Comment by DaanHoogland** [comment]
@pcfriek1987 , I created an issue: #11121 . Can you add version information to it (acs/hypervisor etc)?

**Comment by nvazquez** [comment, marked as answer]
Hi @pcfriek1987 this seems to be the expected behavior for local storage, has been investigated on https://github.com/apache/cloudstack/issues/11121#issuecomment-3078684699
